# Number of footsteps
footstep_num: 16

# Adjacent regularization weight
adjacent_reg_weight: 1e-3

# Whether to switch left and right alternately (supported only in SE2)
# Suppose that the SVM model represents a reachable map from the right foot to the left foot.
alternate_lr: true

# Height of xy plane marker
svm_thre: -0.1

# Limit of configuration update in one step [m], [rad]
delta_config_limit: 0.005

# Initial sample pose
initial_sample_pose:
  translation: [0.0, 0.3, 0.0]

# Whether to predict on grid map
grid_map_prediction: false
